app/views/rewards/_rewards_list.html.erb

```diff
@@ -1,5 +1,8 @@
 <div class="card blue-grey lighten-3 rewards_list container">
 
+<% if current_user == nil %>
```

Comment:
Not sure I've ever seen this syntax before, but usually I'd see "if current_user", and then the code we want them to see, or "if !current_user" then code we don't want them to see.  Might consider updating to be cleaner, but obviously I'm sure this works!